bs.class.php

```diff
@@ -76,10 +84,10 @@ class bs extends game_generic {
                 'parent' => array(
                     'race' => array(
                         0 => 'all',                // Unknown
-                        1 => array(0,2,4,5,9,11),  // Gon
-                        2 => array(0,3,4,7,8,10),  // Lyn
-                        3 => array(0,1,4,5,9,10,12),  // Yun
-                        4 => array(0,1,5,6,8,9,10,11,12),  // Jin
+                        1 => array(0, 2, 4, 5, 9, 11),  // Gon
+                        2 => array(0, 3, 4, 7, 8, 10),  // Lyn
```

Comment:
```suggestion
                        2 => array(0, 3, 4, 6, 7, 8, 10),  // Lyn
```

Reply:
As of this release notes from february, Lyn should be able to be Assassin now. Is this correct?
https://www.bladeandsoul.com/en/news/shroud-of-the-assassin-patch-notes/